notes/PRODUCT.md

```diff
@@ -0,0 +1,32 @@
+# Product Thinking
+The art to defining and solving a problem with a specified target audience, strategy, and goal in mind.
+
+## Layout
```

Comment:
can you explain what this Layout does; flesh out a more descriptive title for the section

Reply:
Updated